index.html

```diff
@@ -30,6 +30,126 @@ <h1>Hey there. I'm John Buhendwa <span>I'm a software developer</span></h1>
                 <li><a href="#"><i class="fa fa-angellist"></i></a></li>
             </ul>
         </section>
+        <div class="button-section">
+            <button><i class="fa fa-angle-down" id="next-section"></i></button>
+        </div>
+        <section class="work-section">
+            <h1>My Recent Works</h1>
+            <div id="hr-line"></div>
+            <div class="project-container">
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
```

Comment:
- [ ] Great job implementing the README file changes requested by the previous reviewer. Please, like she noted, kindly add the hover effect to your `seeproject` and get `my resume` buttons. On hover, the button color should change to light green. This help the user to know he is on the button and thus making your portfolio user-friendly. Please fix that.